doc/source/examples/extended_examples/sfem/sfem.py

```diff
@@ -0,0 +1,758 @@
+
+    # Generate quantities required to define the Young's modulus stochastic process
+    cosine_frequency_list, cosine_eigen_values, cosine_constants = (
+        evaluate_KL_cosine_terms(domain, correl_length_param, min_eigen_value)
+    )
+    sine_frequency_list, sine_eigen_values, sine_constants = evaluate_KL_sine_terms(
+        domain, correl_length_param, min_eigen_value
+    )
+
+    simulation_results = np.zeros(no_of_simulations)
+
+    for simulation in range(no_of_simulations):
+        # Generate random variables and load needed for one realization of the process
+        cosine_random_variables_set = np.random.normal(
+            0, 1, size=len(cosine_frequency_list)
+        )
+        sine_random_variables_set = np.random.normal(
+            0, 1, size=len(sine_frequency_list)
+        )
+        load = -np.random.normal(10, 2**0.5)  # Generate a random load
+
+        material_property = 0  # Initialize material property ID
+        for element_id in element_ids:
+            material_property += 1
+            mapdl.get("ELEMENT_ID", "ELEM", element_id, "CENT", "X")
+            element_centroid_x_coord = mapdl.parameters["ELEMENT_ID"]
+            mapdl.esel(
+                "S", "CENT", "X", element_centroid_x_coord
+            )  # Select all elements having this coordinate as centroid
+
+            # Evaluate young's modulus at this material point
+            young_modulus_value = young_modulus_realization(
+                cosine_frequency_list,
+                cosine_eigen_values,
+                cosine_constants,
+                cosine_random_variables_set,
+                sine_frequency_list,
+                sine_eigen_values,
+                sine_constants,
+                sine_random_variables_set,
+                domain,
+                element_centroid_x_coord,
+            )
+
+            mapdl.mp(
+                "EX", f"{material_property}", young_modulus_value
+            )  # Define property ID, assign young's modulus
+            mapdl.mp(
+                "NUXY", f"{material_property}", poisson_ratio
+            )  # Assign poisson ratio
+            mapdl.mpchg(
+                material_property, "ALL"
+            )  # Assign property to selected elements
+
+        mapdl.allsel()
+
+        mapdl.d("FIXED_END", lab="UX", value=0, lab2="UY")  # Apply fixed end BC
+        mapdl.f("LOAD_APPLICATION_POINT", lab="FY", value=load)  # Apply load BC
+        mapdl.solve()
+
+        # Displacement probe point - where Uy results are to be extracted
+        displacement_probe_point = mapdl.queries.node(length, 0, 0)
+        displacement = mapdl.get_value("NODE", displacement_probe_point, "U", "Y")
+
+        simulation_results[simulation] = displacement
+
+        mapdl.mpdele("ALL", "ALL")
+        if (simulation + 1) % 10 == 0:
+            print(f"Completed {simulation + 1} simulations.")
+
+    mapdl.exit()
+    print()
+    print("All simulations completed!")
```

Comment:
```suggestion
    print("All simulations completed.")
```
Avoid exclamation points in technical doc.